client/src/global-styles/GlobalStyles.tsx

```diff
@@ -7,7 +7,7 @@ const GlobalStyles = createGlobalStyle`
    margin: 0;
 }
 body {
-   font-family: 'Roboto', sans-serif;
+   font-family: 'Poppins', sans-serif;
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;background-color: ${({ theme }) =>
      theme.colors.pageBackground};
```

Comment:
We don't have pageBackground now. you can add the default background color for the application here.